.github/actions/workflow-run-job-linux/action.yml

```diff
@@ -154,8 +164,11 @@ runs:
           --env "GITHUB_WORKSPACE=$GITHUB_WORKSPACE" \
           --env "GITHUB_REPOSITORY=$GITHUB_REPOSITORY" \
           --env "GITHUB_STEP_SUMMARY=$GITHUB_STEP_SUMMARY" \
+          --env "HOST_WORKSPACE=${{github.workspace}}" \
+          --env "NVIDIA_VISIBLE_DEVICES=$NVIDIA_VISIBLE_DEVICES" \
           --volume "${{github.workspace}}/ci.sh:/ci.sh" \
           --volume "${{github.workspace}}/artifacts:/artifacts" \
+          --volume "${{github.workspace}}/wheelhouse:/home/coder/cccl/wheelhouse" \
```

Comment:
I don't like that I have to hardcode the prefix `/home/coder/cccl/` here, and alternative solutions are welcome.

Mainly, I need the `wheelhouse` directory to live _inside_ the `cccl` source directory as I want the wheel-building scripts to be executable locally, and in that case I don't want them to mess with the filesystem outside of the `cccl/` directory.

Reply:
The entire `cccl` dir is mounted to `/home/coder/cccl` , so couldn't you do `mkdir -p "${{github.event.repository.name}}/wheelhouse"` on L71? I think that'd still be accessible to the `Upload wheelhouse` step below.

Reply:
Also if it's expected the scripts are runnable from within the devcontainer and `wheelhouse` is the output dir, you could update the init script to create the `wheelhouse` dir on the host ([line 10](https://github.com/NVIDIA/cccl/blob/2f6a8e295c4d761fd9f2eed7aa8157e01720ee51/.devcontainer/devcontainer.json#L10C68-L10C114) in the `devcontainer.json` files) like this:
```diff
- mkdir -m 0755 -p ${localWorkspaceFolder}/build
+ mkdir -m 0755 -p ${localWorkspaceFolder}/{build,wheelhouse}
```

Then you wouldn't need to manually create the dir in here at all, it would be created by the devcontainers CLI/extension locally, or `launch.sh` in CI.

Reply:
I think that all would work. Wouldn't I need something similar in the `launch_docker` function in `launch.sh` as well? For those running locally but not on vscode?

Reply:
`launch.sh` [executes](https://github.com/NVIDIA/cccl/blob/6983639b377724a872286078f0533b7bf941778c/.devcontainer/launch.sh#L240-L243) the devcontainer.json's `initializeCommand`, so whatever's in there should work for both local dev and `launch.sh` usage.

Reply:
Fixed, after some whack-a-mole. thank you!